### PR TITLE
Axis fixes

### PR DESCRIPTION
**Pull request by ldecicco-USGS**

### CODE REVIEW

File: R/access-gsplot.R [hunk 1]
```diff
@@ -72,4 +72,28 @@ logged.gsplot <- function(object, side=NULL){
...
+}
+
+#' Get view information from a gsplot object
+#' 
+#' get the views in gsplot object
+#' 
+#' @param object a gsplot object
+#' @return data frame with one row per view. Each view has an x side, y side, the log command, and the view index.
+#' @export
+getViewInfo <- function(object){
```
Comment (ldecicco-USGS) [inline R/access-gsplot.R]: @jread-usgs check out this function and tell me if I re-invented some other function

Comment (jordansread) [inline R/access-gsplot.R]: no, we don't have something like this. `views(gsplot)` just returns the views (removes the other top level things like par and legend).

Comment (ldecicco-USGS) [inline R/access-gsplot.R]: cool. It's kind of handy, might be useful for more than just axis.

Comment (jordansread) [inline R/access-gsplot.R]: camelCase..?

Comment (ldecicco-USGS) [inline R/access-gsplot.R]: Sorry, hard habit to break. I'll quick change it. We are doing _ for functions and . for arguments, right?

**Comment by ldecicco-USGS** [comment]
OK, the pressure to not reduce the coverage is great. Anyway, I'm done. If someone wants to poke around before merging, cool. Otherwise, you can play with the `axis` examples to see if it acts the way you think it should act.

File: README.Rmd [hunk 2]
```diff
@@ -54,7 +54,8 @@ gs <- gsplot() %>%
   lines(c(3,4,3), c(2,4,6), legend.name="Lines", ylab="logged y axis", xlab="logged x axis", log='xy') %>%
   callouts(x=8, y=4, lwd=2, angle=45, labels="Not sure about this one") %>%
   title("logged axes") %>%
-  axis(side=c(1,2,3,4), labels=FALSE, n.minor=4)
+  axis(side=c(1,2,3,4), labels=FALSE, n.minor=4) %>%
+  axis(side=c(1,2))
```
Comment (jordansread) [inline README.Rmd:58]: looks like `grid` happens on top of axis(side=1) in `unnamed-chunk-2-1.png`, which is why there is a diff for that file. Odd that it doesn't matter for side=3.

File: README.md [hunk 3]
```diff
@@ -51,7 +51,8 @@ gs <- gsplot() %>%
   lines(c(3,4,3), c(2,4,6), legend.name="Lines", ylab="logged y axis", xlab="logged x axis", log='xy') %>%
   callouts(x=8, y=4, lwd=2, angle=45, labels="Not sure about this one") %>%
   title("logged axes") %>%
-  axis(side=c(1,2,3,4), labels=FALSE, n.minor=4)
+  axis(side=c(1,2,3,4), labels=FALSE, n.minor=4) %>%
+  axis(side=c(1,2))
 gs
```
Comment (jordansread) [inline README.md:56]: :+1: on `unnamed-chunk-3-1.png` minor tick thingy

**Comment by jordansread** [comment]
Looks great to me. I'd say :+1: for merge

File: README.Rmd (hunk 2 above)
Comment (ldecicco-USGS) [inline README.Rmd:58]: I think it's actually due to not ordering the views anymore. We use to put the bgCol first, then grid, then the rest. I'm planning to move those to a top level thing like axis and par (see #239 ) .

**Pull request merged**